mvc/src/main/java/com/interface21/webmvc/servlet/mvc/tobe/HandlerExecution.java

```diff
@@ -15,7 +16,13 @@ public HandlerExecution(Object controller, Method method) {
         this.method = method;
     }
 
-    public ModelAndView handle(final HttpServletRequest request, final HttpServletResponse response) throws Exception {
-        return (ModelAndView) method.invoke(controller, request, response);
+    public ModelAndView handle(final HttpServletRequest request, final HttpServletResponse response) {
+        try {
+            return (ModelAndView) method.invoke(controller, request, response);
+        } catch (IllegalAccessException e) {
+            throw new RuntimeException(e);
+        } catch (InvocationTargetException e) {
+            throw new RuntimeException(e);
+        }
```

Comment:
```suggestion
try {
    return (ModelAndView) method.invoke(controller, request, response);
} catch (IllegalAccessException | InvocationTargetException e) {
    throw new RuntimeException(e);
}
```

두 타입에 대한 예외 처리가 동일하면 하나의 catch 블록으로 묶어도 될 것 같아요~ 그리고 이 부분에도 method에 대한 로깅을 추가하거나 예외 메시지를 더 다듬어볼수도 있겠습니다!

Reply:
별도의 예외로 처리해주고 싶은데 아직 예외에 대한 세부적인 부분은 고민하지 않았어요!
우선 레벨4가 너무 빠쁜 관계로 기능 구현에 중점을 두고 STEP3 이후에 개인적으로 아쉬웠던 처리에 대해서 개선해보려고 해요!
그래도 좋은 의견 제시 감사합니다~! 다음에는 이런 포인트가 있다면 리뷰 요청시에 같이 말씀드릴게요 ㅎㅎ

Reply:
> 별도의 예외로 처리해주고 싶은데 아직 예외에 대한 세부적인 부분은 고민하지 않았어요! 우선 레벨4가 너무 빠쁜 관계로 기능 구현에 중점을 두고 STEP3 이후에 개인적으로 아쉬웠던 처리에 대해서 개선해보려고 해요! 그래도 좋은 의견 제시 감사합니다~! 다음에는 이런 포인트가 있다면 리뷰 요청시에 같이 말씀드릴게요 ㅎㅎ

넵 ㅎㅎ 3단계때는 천천히 고민해볼 코드에 대해서는 주석이나 별도의 커멘트 남겨주시면 참고할게요 ~!